pkg/products/amqonline/reconciler.go

```diff
@@ -114,7 +115,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 	operatorNamespace := r.Config.GetOperatorNamespace()
 	productNamespace := r.Config.GetNamespace()
 
-	phase, err := r.ReconcileFinalizer(ctx, serverClient, installation, string(r.Config.GetProductName()), uninstall, func() (integreatlyv1alpha1.StatusPhase, error) {
+	phase, err := r.ReconcileFinalizer(ctx, serverClient, installation, string(r.Config.GetProductName()), rhmiv1alpha1.MonitoringStage, uninstall, func() (integreatlyv1alpha1.StatusPhase, error) {
```

Comment:
@valerymo the Stage passsed by each reconciler should be it's own one.

Reply:
Although now that i look at it a bit deeper. there is no stage for AMQOnline for example. This gets included in the ProductsStage.

Reply:
@laurafitzgerald   Stage passsed by each reconciler Changed to be it's own one. The question just about those that using `ProductsStage` - if it's ok, I set it as `ProductsStage`, that is same that using function `HandleProductComplete(..,)`. Is it ok?
One more question - about Observability reconciler. I used here - `integreatlyv1alpha1.ObservabilityStage`, although in   `HandleProductComplete(..,)` - integreatlyv1alpha1.ProductsStage